### DISCUSSION TITLE
Vontar X4 not booting from SD

### DISCUSSION THREAD

**Discussion by albarreto** [devmfc/debian-on-amlogic#58] [General]
Hi,

I installed devmfc bookworm on TX3 Mini+ with success.
But now I'm trying to install in my newest TV Box Vontar X4, but I cant boot from SD.
I followed the steps presssing reset button (inside AV hole) and power on, holding for 10 secs the reset.
The screen freezes with orange Vontar logo.

Tested with Devmfc_Debian-Bookworm_6.6.29-meson64_Minimal-24.04.29.img.xz
Config.boot with "box=vontarx4_gigabit" uncommented

[UPDATE]
With old release Devmfc_Debian-Bookworm_6.6.8-meson64_Minimal-23.12.22.img.xz, IT WORKS!

Any idea?

**Comment by devmfc** [comment]
What box config did you use with the older image? I think that Vontar X4 was not even there then?

**Reply by jojkol** [reply]
> 您是否已使用过哪个盒子配置？ 我认为当时 Vontar X4 还会有故障吗？

My box is Amlogic s905x4. After I used your method to enter the system, I always encountered command not found when using the apt command. I tried various methods but could not solve this problem. I suspect it is a problem with the software source, so I would like to ask you, in which file is the software source stored?

**Comment by evandrojr** [comment]
I have just booted Vontar X4 with Ubuntu from SDCARD. Works great except for desktop instalation that didn´t work.

**Reply by petesimon** [reply]
there are many desktop environments. which one (or more) did you try ?

**Comment by petesimon** [comment]
I have a Vontar X4 and Android 11, 20230816.091701, and Kernel 5.4.125 Wed Aug 16 2023 was included on the box. I installed "Devmfc_Ubuntu-Oracular_6.6.60-meson64_Minimal-24.11.10.img" to a 64GB SD card, un-commented "box=vontarx4_gigabit" in the config file, put the card in the slot, held the reset button, connected power and boom it booted right up. I checked ethernet with `ethtool eth0` and there is gigabit speed. awesome!

Regarding [3rd party OS's _not_ booting](https://github.com/ophub/amlogic-s9xxx-armbian/issues/2259) on a [H96 Max X3 (S905x3)](https://archive.org/details/H96Max-X3-s905x3-tvbox-android-firmware-backup) because of it having a 20230208 dated stock Android firmware, I can guesstimate that some other tv boxes may also have the same symptom. So, maybe owners of Vontar X4 could flash 20230816.091701 or older Android firmware to enable booting a 3rd party OS.. Just a thought 🤓
